sycl/plugins/level_zero/usm_allocator.cpp

```diff
@@ -673,6 +686,14 @@ void *Slab::getEnd() const {
 
 bool Slab::hasAvail() { return NumAllocated != getNumChunks(); }
 
+// If a slab was available in the pool then note that the current pooled
+// size has reduced by the size of this slab.
```

Comment:
> the size of this slab

What is this slab?

Reply:
A slab in this bucket is being allocated out of the pool. When this happens we call decrementPool to update statistics about the pool. The pool size has just reduced by one slab in this bucket, i.e. the size of a slab in this bucket. 

I've updated the comment to say "reduced by the size of a slab in this bucket."